get_authors.rb

```diff
@@ -12,15 +17,9 @@ def md_center(text)
 files = Dir.glob File.join('*', '**', '*.md')
 
 files.each do |f|
-  author_mail = {}
-  author_md = []
-  command = 'git log -5 --pretty=format:"%an;%ae" '
-  command_out = %x(#{command}#{f}).split(/[;\n]/)
-  author_mail = command_out.each_slice(2).to_h.sort
-
-  author_mail.each do |a, e|
-    author_md << md_mailto(a, e)
-  end
+  author_mail = get_authors(f).each_slice(2).to_h.sort
```

Comment:
I do still see some `magic` in here. As this is only refactoring, and the line was not changed, I do not object.